src/frontend/src/components/Post/Post.jsx

```diff
@@ -93,6 +92,10 @@ const Post = ({ postUrl }) => {
   const sectionEl = useRef(null);
   // Grab the post data from our backend so we can render it
   const { data: post, error } = useSWR(postUrl, (url) => fetch(url).then((r) => r.json()));
+  const [expandToggle, setToggle] = React.useState(true);
```

Comment:
`expandHeader` and `setExpandHeader` would be better.  Also, import `useState` at the top of this file vs. using `React.*`.